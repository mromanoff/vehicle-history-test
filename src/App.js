import React, { Component } from 'react';
import './styles/main.css';

import Header from './components/Header';
import Footer from './components/Footer';
import Categories from './components/Categories';
import Gallery from './Gallery';
import Gallery2 from './Gallery2';
import { CATEGORIES } from './constants';

class App extends Component {
  render() {
    return (
      <div className="application">
        <Header />
        <main className="container">
          {/*<Gallery initialQuery="dog"/>*/}
          <Gallery2
            initialCategory="Animals"
            initialPage="1"
            categories={CATEGORIES}
          />
          {/*<Categories initialCategory="Animals" />*/}
        </main>
        <Footer />
      </div>
    );
  }
}

export default App;
