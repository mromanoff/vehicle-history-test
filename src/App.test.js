import React from 'react';
import App from './App';

it('renders app', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
