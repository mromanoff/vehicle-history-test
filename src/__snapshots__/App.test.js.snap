// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<div
  className="application"
>
  <Header />
  <main
    className="container"
  >
    <Categories
      categories={
        Array [
          "Modeling",
          "Animals",
          "Transportation",
          "Nature",
          "Sport",
        ]
      }
      initialCategory="Animals"
      initialPage={1}
      itemsCountPerPage={20}
    />
  </main>
  <Footer />
</div>
`;
